stablehlo/reference/Ops.cpp

```diff
@@ -102,6 +102,94 @@ Tensor evalSliceOp(const Tensor &operand, const Sizes &startIndices,
                      inferredTypes[0].cast<ShapedType>());
 }
 
+template <typename T>
+DenseIntElementsAttr getDenseIntElementsAttr(
+    Type elementType, T values,
+    std::optional<SmallVector<int64_t>> valuesShape) {
+  SmallVector<int64_t> shape =
```

Comment:
It seems that you can trying to generalize this function. In the current use case, we can specialize it only for 1-D tensor for dimensions, right?